nucypher/cli/commands/stake.py

```diff
@@ -948,3 +960,48 @@ def set_min_rate(general_config, transacting_staker_options, config_file, force,
                           receipt=receipt,
                           chain_name=blockchain.client.chain_name,
                           transaction_type='set_min_rate')
+
+
+@stake.command()
+@group_transacting_staker_options
+@option_config_file
+@option_force
+@group_general_config
+def mint(general_config, transacting_staker_options, config_file, force):
+    """Mint last portion of reward"""
```

Comment:
It might be helpful to include thew command somewhere in the greater documentation:  When is usage of this command indicated for a  node operator?

Reply:
I've added line about mint in staking guide

Reply:
I noticed, however it;s pretty terse.  Something verbosity about when usage is indicated will be more helpful.

Reply:
any suggestions?

Reply:
Two Suggestions
-------------------------
- *When* does a node operator need to use this command?
- Update CLI command documentation table

Reply:
✔️